code/modules/reagents/chemistry/reagents.dm

```diff
@@ -23,6 +23,7 @@ GLOBAL_LIST_INIT(name2reagent, build_name2reagent())
 	var/glass_name = "glass of ...what?" // use for specialty drinks.
 	var/glass_desc = "You can't really tell what this is."
 	var/glass_icon_state = null // Otherwise just sets the icon to a normal glass with the mixture of the reagents in the glass.
+	var/glass_icon_file = "icons/obj/drinks.dmi" // monkeystation addition
```

Comment:
This is going to have the same issue as your other drink PR.
You're not actually using this anywhere.

Reply:
i was told to add this bc there doesnt seem to be a icon variable for the drinks and you would complain if i put it into icons/obj/drinks.dmi